commands/paste-emoji/command.toml

```diff
@@ -0,0 +1,16 @@
+name = "Paste Emoji"
```

Comment:
Let's name it "Search Emoji"? 

"Paste" is only 1 action that your command can allow to do. It can also let you do other things, e.g. Copy.